packages/configuration-builder/src/ColorsSection/BrandColors.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, Headline, Inline, Stack } from "@buildo/bento-design-system";
+import { useTranslation } from "react-i18next";
+import { ColorConfig, ColorEditor } from "../ColorEditor/ColorEditor";
+import { HexColor } from "../utils/colorUtils";
+import { ThemeConfig } from "../ThemeConfigurator/ThemeConfigurator";
+import { defaultColorConfig } from "./defaultColor";
+
+type BrandColors = ThemeConfig["colors"]["brand"];
+
+type Props = {
+  value: BrandColors;
+  onChange: (value: BrandColors) => void;
+};
+
+const defaultColor: ColorConfig = defaultColorConfig("#7F77FF" as HexColor);
+
+export function BrandColors(props: Props) {
+  const { t } = useTranslation();
+
+  function onAddBrandColor() {
+    switch (props.value.length) {
+      case 1:
+        return props.onChange([props.value[0], defaultColor]);
+      case 2:
+        return props.onChange([props.value[0], props.value[1], defaultColor]);
+    }
```

Comment:
Can't `BrandColor` have also 3 values?

Maybe we can use `ts-pattern` to get an exhaustive check?